### ISSUE TITLE
Documentation links are broken

### ISSUE THREAD

**Issue by joachimbrand** [Jutho/VectorInterface.jl#18]
Thanks for this very nice package!

I noticed that the documentation links in the `README.md` file are broken.

**Comment by Jutho** [comment]
Yes, I should at some point convert the README into an actual documentation and update it, though in principle everything is in the README. I will leave this open so that I have an occasional reminder, but at the moment there are things on my plate with higher priority.
